src/pybind/mgr/prometheus/module.py

```diff
@@ -508,6 +517,16 @@ def get_metadata_and_osd_status(self):
         for pool in osd_map['pools']:
             self.metrics.append('pool_metadata', 1, (pool['pool'], pool['pool_name']))
 
+        # Populate rgw_metadata
+        for server in servers:
```

Comment:
Rather than doing more `[]` on servers inside the loop, I'd just do a `for svc_key, svc_data in servers.items()` here and then do a `svc_id, svc_type = svc_key` and `hostname, version = svc_data` inside the loop.  Named variables will make it much easier to read.